src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -109,14 +111,14 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
       this.loading = true
 
       const isOauth = this.signInLocal.type === TypeSignIn.oauth
-      const $signIn = this._signIn.signIn(this.signInLocal, isOauth)
+      const $signIn = this._signIn.signIn(this.signInLocal, isOauth, isOauth)
       $signIn.subscribe((data) => {
         this.loading = false
         this.printError = false
 
         if (data.success) {
-          if (data.url.toLowerCase().includes('oauth/authorize')) {
-            this.updateOauthSession(this.signInLocal.params)
+          if (isARedirectToTheAuthorizationPage(data)) {
```

Comment:
I think we should respect the camel case and it will be more readable to rename the method to `IsRedirectToAuthorizationPage` or something similar